webhookGitHubComment/index.js

```diff
@@ -82,6 +83,16 @@ module.exports = azureWrapper(async function webhookGitHubComment(context, req)
         }, 
       ]
     }, { headers: { authorization: `Bearer ${config.slackToken}` } });
+  } else if (action === 'created') {

```

Comment:
Make this `else if (action === 'created' && comment != null)`

I wouldn't be surprised if GitHub sends `action: 'created'` if you create something other than a comment, like a tag or a release.